src/lava_callback.py

```diff
@@ -66,6 +66,9 @@ def async_job_submit(api_helper, node_id, job_callback):
     '''
     results = job_callback.get_results()
     job_node = api_helper.api.node.get(node_id)
+    if not job_node:
+        print(f'Node {node_id} not found')
```

Comment:
Maybe add a bit more context? like `Node <id> not found for callback (<other relevant information if available>)`

Reply:
It is quite exceptional case, it happened with me only when i manually edited LAVA job with random node id, just need small notification if such abnormal thing will happen and to prevent lava-callback crashing on it.